### PR TITLE
Rework the Filecoin<->IPFS page

### PR DESCRIPTION
**Pull request by hsanjuan**

### CODE REVIEW

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 1]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
+- IPFS by itself works very well for popular content (with many providers), for organizations where there are incentives to sync and store data in multiple nodes and for situation where strong social contracts can be used to ensure the content remains hosted and maintained long-term.
 
 ### Using Filecoin
 
-The final option is to pin your data to a decentralized storage marketplace, such as Filecoin. In Filecoin's structure, clients make regular small payments to store data at a specified availability. Simultaneously, miners earn these payments by continuously verifying this data's integrity, storing it, and ensuring it can be retrieved quickly. This allows users to incentivize Filecoin miners to ensure their content will be live when it's needed, a clear advantage over depending solely on the generosity of other network users as required with using IPFS on its own.
+- Clients make deals with miners to store data. The network verifies that the data is indeed stored by the miners, and regular, small payments are made for the duration of the deal.
```
Comment (johnnymatthews) [inline docs/about-filecoin/ipfs-and-filecoin.md:32]:
```suggestion
- Clients make _storage deals_ with miners to store data. The network verifies that the miners are correctly storing the data. Small payments are made on a regular basis for the duration of the _storage deal_.
```

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 2]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
+- Miners that are unable to honor the storage conditions they offered have to pay a penalty.
```
Comment (johnnymatthews) [inline docs/about-filecoin/ipfs-and-filecoin.md:33]:
```suggestion
- Miners that do not honor the storage deal are penalised.
```

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 3]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
+- Content retrieval might be offered by storage miners directly or by specialized retrieval miners, for a price.
```
Comment (johnnymatthews) [inline docs/about-filecoin/ipfs-and-filecoin.md:34]:
```suggestion
- Content retrieval might be offered by storage miners directly, or by specialized retrieval miners. The user requesting the data pays for this service.
```

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 4]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
+- Filecoin by itself work very well for large amounts of data to be stored for longer periods of time.
```
Comment (johnnymatthews) [inline docs/about-filecoin/ipfs-and-filecoin.md:35]:
```suggestion
- Filecoin excels at storing large amounts of data for long periods of time.
```

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 5]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
+
+### Using both
 
-## Filecoin, powered by IPFS
+Some solutions like [Powergate](../build/powergate.md) combine the best of the two worlds, backing up data on the Filecoin network and at the same time providing it through the IPFS network. This ensures availability and fast retrieval while knowing, at the same time, that the data is safely backed up by one or several Filecoin miners.
```
Comment (johnnymatthews) [inline docs/about-filecoin/ipfs-and-filecoin.md:39]:
```suggestion
Some solutions combine the best of the to systems, backing up data on the Filecoin network and at the same time providing the data through the IPFS network. This ensures that data is constantly available and can be retrieved quickly, while also making sure that the data is safely backed up on the Filecoin network. [Powergate](../build/powergate.md) is a service that does this.
```

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 6]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
-- Messaging between nodes and block propagation in Filecoin are facilitated by [libp2p](https://libp2p.io/) pubsub
-- CIDs in Filecoin and IPFS share a hashing specification
-- The use of [graphsync](https://github.com/ipfs/go-graphsync) for transferring data between nodes, enabling direct transfers between IPFS and Filecoin nodes in the future.
+- [IPLD](https://ipld.io/) specifies data formats for content-addressed data like the blockchain or the way in which IPFS stores files.
+- [libp2p](https://libp2p.io/) provides peer-to-peer network capabilities, connection security and key discovery and data distribution features like the DHT and Pubsub.
+- [Multiformats](https://multiformats.io) define future-proof identifiers and data-types.
+- [Graphsync](https://github.com/ipfs/go-graphsync) and [Bitswap](https://github.com/ipfs/go-bitswap) enable fast an efficient IPLD data transfers between nodes.
 
-**Interested in learning more?** Check back here soon for various use-case examples featuring both protocols!
+**Interested in learning more?** Check some of our [application examples](../build/examples/README.md).
```
Comment (johnnymatthews) [inline docs/about-filecoin/ipfs-and-filecoin.md:50]:
```suggestion
Interested in learning more? Check some of our [application examples](../build/examples/README.md).
```

**Review by johnnymatthews — CHANGES_REQUESTED** [review]

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 7]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
+## Data storage incentives
 
-The lack of a built-in incentive mechanism is the challenge Filecoin hopes to solve by allowing users to incentivize long-term distributed storage at competitive prices through a marketplace of storage nodes while maintaining the efficiency and resiliency provided by the IPFS network.
+[IPFS](https://ipfs.io) allows users to store and transfer verifiable, content-addressed data in a peer-to-peer network. IPFS users persist the data they want **on their own IPFS nodes**. This is referred as [pinning](https://docs.ipfs.io/concepts/persistence). Sometimes the data may be pinned using a third-party pinning service, or through groups of individual IPFS users. The data exists in the network as long as one user is storing it and able to provide it to others when they request it.
```
[the remark this answers is not in the document]
Comment (hsanjuan) [reply docs/about-filecoin/ipfs-and-filecoin.md]: There are subtleties here. `can provide it to others when requested` is ambiguous as it does not specify who requests it. It would seem as if the node storing it needs to request it.

I also don't fully understand why we remove all bold highlighting. There has to be a way to make certain key pieces of information stand out in a paragraph, it is not something we abuse.

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 8]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
 
-## Different persistence strategies: which should I use?
+However, IPFS does not include a built-in mechanism to incentivize the storage of data for _other_ people. This is the challenge Filecoin hopes to solve: the Filecoin network creates a distributed storage market place for long-term storage where nodes with large storage capacity can rent it to users with data-storage needs, and get paid for it.
 
-Depending on your use case, one or multiple of these persistence strategies might be a good fit for you.
+The cryptographic guarantees ensuring that data is safely stored by the Filecoin network make the storage and retrieval of data a computationally and time-expensive operation. For this reason, Filecoin enables an additional retrieval market where dedicated nodes can help quickly delivering content from the network for a payment. This delivery mechanism may make use of IPFS.
```
[the remark this answers is not in the document]
Comment (hsanjuan) [reply docs/about-filecoin/ipfs-and-filecoin.md]: This changes the meaning. The cryptography is not to ensure that data is retreivable. 

It is very different to say that "the crytography guarantees around storing and retrieving content are expensive" and that "the data is cryptographially guaranteed so that it is retrievable". It is nuanced but this editing departs from the original meaning.

File: docs/about-filecoin/ipfs-and-filecoin.md [hunk 9]
```diff
@@ -5,38 +5,46 @@ description: Learn more about the relationship and different use-cases between I
...
 
-### Using IPFS
+In that sense, Filecoin can be seen as a "cold" storage layer, perfect to safely store large batches of data, while IPFS would be the "hot" storage layer, designed for the quick retrieval and distribution of content.
+
+## Which should I use?
```
[the remark this answers is not in the document]
Comment (hsanjuan) [reply docs/about-filecoin/ipfs-and-filecoin.md:22]: Better systems. They are not single protocols. Protocol suites, if anything, and still it is complex because of how loaded the word "protocol" is in network architectures. Therefore I try to stay away from it.

File: docs/about-filecoin/ipfs-and-filecoin.md (hunk 5 above)
Comment (hsanjuan) [reply docs/about-filecoin/ipfs-and-filecoin.md]: s/to/two

Comment (hsanjuan) [reply docs/about-filecoin/ipfs-and-filecoin.md]: It sounds weird to say powergate is a service. It's a solution, or a system, or an application, or an integration, sometimes offered as a service.

File: docs/about-filecoin/ipfs-and-filecoin.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (johnnymatthews) [reply docs/about-filecoin/ipfs-and-filecoin.md]: This is a _very_ wordy sentence that is difficult to follow:

> The cryptographic guarantees ensuring that data is safely stored by the Filecoin network make the storage and retrieval of data a computationally and time-expensive operation.

My commit was an attempt to make things easier to follow. If I've changed the meaning, however, can you try splitting the above sentence into two. That'd make this section clearer, without changing the meaning of the paragraph.

File: docs/about-filecoin/ipfs-and-filecoin.md (hunk 5 above)
Comment (johnnymatthews) [reply docs/about-filecoin/ipfs-and-filecoin.md]: The repo for Powergate reads:

> Powergate is a multitiered file storage API built on Filecoin and IPFS, and and index builder for Filecoin data. It's designed to be modular and extensible.

I'll just use that.

**Review by johnnymatthews — APPROVED** [review]

File: docs/about-filecoin/ipfs-and-filecoin.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (hsanjuan) [reply docs/about-filecoin/ipfs-and-filecoin.md]: > This is a _very_ wordy sentence that is difficult to follow

I agree! I'll reword it tomorrow. Thanks for flagging!
